jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/displaydata/AvatarTests.java

```diff
@@ -1,78 +1,44 @@
+import java.util.List;
 
-public class AvatarTests extends TestsInit {
-    @Test
-    public void noImageTest(){
-        dataDisplayNoImageAvatarPage.open();
-
-        dataDisplayAvatarFrame.noImage.is().displayed();
-        dataDisplayAvatarFrame.noImage.is().hasClass("MuiAvatar-circle");
-        dataDisplayAvatarFrame.noImage.is().hasClass("MuiAvatar-colorDefault");
-    }
-
-    @Test
-    public void circleTest(){
-        dataDisplayCircleAvatarPage.open();
-
-        dataDisplayAvatarFrame.circle.is().displayed();
-        dataDisplayAvatarFrame.circle.is().hasClass("MuiAvatar-circle");
-        dataDisplayAvatarFrame.circle.is().displayedImg();
-        dataDisplayAvatarFrame.circle.is().hasImg("MuiAvatar-img");
-    }
-
-    @Test
-    public void squareTest(){
-        dataDisplaySquareAvatarPage.open();
+import static io.github.com.pages.displaydata.AvatarPage.onlineStatus;
+import static io.github.com.pages.displaydata.AvatarPage.avatarWithPhoto;
+import static io.github.com.pages.displaydata.AvatarPage.avatarWithoutPhoto;
 
-        dataDisplayAvatarFrame.square.is().displayed();
-        dataDisplayAvatarFrame.square.is().hasClass("MuiAvatar-square");
-        dataDisplayAvatarFrame.square.is().displayedImg();
-        dataDisplayAvatarFrame.square.is().hasImg("MuiAvatar-img");
-    }
-
-    @Test
-    public void roundedTest(){
-        dataDisplayRoundedAvatarPage.open();
-
-        dataDisplayAvatarFrame.rounded.is().displayed();
-        dataDisplayAvatarFrame.rounded.hasClass("MuiAvatar-rounded");
-        dataDisplayAvatarFrame.rounded.is().displayedImg();
-        dataDisplayAvatarFrame.rounded.is().hasImg("MuiAvatar-img");
-    }
 
-    @Test
-    public void fallbackAltTest(){
-        dataDisplayFallbackAltAvatarPage.open();
 
-        dataDisplayAvatarFrame.fallbackAlt.is().displayed();
-        dataDisplayAvatarFrame.fallbackAlt.is().hasClass("MuiAvatar-circle");
-        dataDisplayAvatarFrame.fallbackAlt.is().hasClass("MuiAvatar-colorDefault");
-        dataDisplayAvatarFrame.fallbackAlt.is().text("L");
+public class AvatarTests extends TestsInit {
+    @BeforeTest
+    public void beforeTest() {
+        //openSection("Avatar");
+        StaticSite.dataDisplayAvatarPage.open();
+        StaticSite.dataDisplayAvatarPage.isOpened();
     }
 
     @Test
-    public void letterTest(){
-        dataDisplayLetterAvatarPage.open();
-
-        dataDisplayAvatarFrame.letter.is().displayed();
-        dataDisplayAvatarFrame.letter.is().hasClass("MuiAvatar-circle");
-        dataDisplayAvatarFrame.fallbackAlt.is().hasClass("MuiAvatar-colorDefault");
-        dataDisplayAvatarFrame.fallbackAlt.is().text("A");
+    public void avatarTests() {
+        testLogic(avatarWithoutPhoto);
+        testLogic(avatarWithPhoto);
+        onlineStatus.get(1).has().classValue(Matchers.containsString("MuiBadge-dot"));
+        onlineStatus.get(2).has().classValue(Matchers.containsString("MuiBadge-anchorOriginBottomRightCircle"));
```

Comment:
Please also check color and text of the avatar, like it was done in previous tests

Reply:
Added check for the color and text where it's applicable.